prbt_hardware_support/test/integrationtests/integrationtest_brake_test_required.cpp

```diff
@@ -71,7 +71,7 @@ class BrakeTestRequiredIntegrationTest : public testing::Test, public testing::A
 
 ::testing::AssertionResult expectBrakeTestRequiredServiceCallResult
                                              (ros::ServiceClient& brake_test_required_client,
-                                              bool expectation,
+                                              IsBrakeTestRequiredResponse::_result_type expectation,
                                               uint16_t retries)
```

Comment:
it always 10, anyway. Please introduce `uint16_t retries = DEFAULT_RETRIES` (as in the unittest)